tests/sles4sap/qesapdeployment/destroy.pm

```diff
@@ -13,6 +13,11 @@ use sles4sap::qesap::qesapdeployment;
 
 sub run {
     select_serial_terminal;
+
+    if (check_var('PUBLIC_CLOUD_PROVIDER', 'EC2') && get_var("QESAPDEPLOY_IBSMIRROR_IP_RANGE")) {
```

Comment:
Single quotes? Please, please please? :D
```
"QESAPDEPLOY_IBSMIRROR_IP_RANGE"
```

Sorry... that's somehow my thing now.